src/main/java/com/softserve/academy/spaced/repetition/service/UserService.java

```diff
@@ -225,27 +226,32 @@ public User uploadImage(MultipartFile file) throws ImageRepositorySizeQuotaExcee
 
     public void activateAccount() throws NotAuthorisedUserException {
         User user = getAuthorizedUser();
-        user.getAccount().setStatus(AccountStatus.ACTIVE);
+        user.getAccount().setDeactivated(false);
         mailService.sendConfirmationMail(user);
         userRepository.save(user);
     }
 
     public void deleteAccount() throws NotAuthorisedUserException {
         User user = getAuthorizedUser();
-        user.getAccount().setStatus(AccountStatus.INACTIVE);
+        user.getAccount().setDeactivated(true);
         userRepository.save(user);
     }
 
     public void getUserStatus() throws UserStatusException {
         JwtUser jwtUser = (JwtUser) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
-        User user = userRepository.findUserByAccountEmail(jwtUser.getUsername());
-        if (user.getAccount().getStatus().isNotActive()) {
-            throw new UserStatusException(user.getAccount().getStatus());
+        System.out.println(jwtUser.isAccountNonLocked());
```

Comment:
Please use LOGGER instead of System.out. Sonar should report this.

Reply:
oops, it should not be here. I'll fix it thanks